src/pmsearch/pmsearch_index.sh

```
@@ -0,0 +1,235 @@
+    if $VERBOSE
+    then
+	echo "$prog: no runtime data to add"
+    fi
+    status=0
+    exit
+fi
+
+# Add/update all runtime data (metrics and instances) into the index.
+# Metrics already in the base index are updated; new metrics from
+# PMDAs without help files (e.g. Python PMDAs) are inserted.
+if $PCP_BINADM_DIR/newhelp -S -o "$NEW" $tmp/helptext && mv -f "$NEW" "$INDEX"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)